gatt/gatt_linux.py

```diff
@@ -596,21 +596,28 @@ def read_value(self, offset=0):
             error = _error_from_dbus_error(e)
             self.service.device.characteristic_read_value_failed(self, error=error)
 
-    def write_value(self, value, offset=0):
+    def write_value(self, value, offset=0, with_response=True):
         """
         Attempts to write a value to the characteristic.
 
         Success or failure will be notified by calls to `write_value_succeeded` or `write_value_failed` respectively.
 
         :param value: array of bytes to be written
         :param offset: offset from where to start writing the bytes (defaults to 0)
+        :param with_response: whether a response should be requested for the write (defaults to True)
         """
         bytes = [dbus.Byte(b) for b in value]
 
+        if with_response == True:
+            write_type = "request"
+        else:
+            write_type = "command"
+
         try:
             self._object.WriteValue(
                 bytes,
-                {'offset': dbus.UInt16(offset, variant_level=1)},
+                {'offset': dbus.UInt16(offset, variant_level=1),
+                 'type': write_type},
```

Comment:
Thanks for your PR! It's important that we don't break existing projects using this library. Since this parameter was previously not specified, what is the default value of `type` if omitted?

Reply:
As I read the PR, the PR proposes the `write_value()` function to receive a new optional parameter `with_response=True` and thus existing code not providing that explicitly will get the default of True; in turn that will set `write_type` to `"request"` and that is then given to `type` if omitted.
So, that is the long story; and in short: existing code would with this PR default to `'type': "request"` (as they will default to the `with_response=True` parameter.